### ISSUE TITLE
Test DeepSeg on SCI patients from UCSF 

### ISSUE THREAD

**Issue by SaraDupont** [neuropoly/spinalcordtoolbox#1659]
Testing new function `sct_deepseg_sc` on acute trauma patients from UCSF (42 SCI patients).
--> tested with SCT v3.1.1

- One issue when trying to segment the full image (see screenshot), issue does not happen when segmenting an image cropped around SC
![error_not_crop](https://user-images.githubusercontent.com/5857909/37920085-bbed4724-30da-11e8-8fd5-f6faf2c41b72.png)

- Really good results overall (see below) : 
      - DC against manual segmentations  (mean +- std [min, max]) = 0.84+-0.09 [0.39, 0.93] 
      - When removing the min and max value --> 0.86+-0.07 [0.60, 0.92]

- qualitatively: some leaks and imperfections at the level of compression

- only one patient did really poorly --> strong compressions and change of contrast around SC (patient 1002)

``` 
         DC patient
0   0.900318    1050
1   0.793758    1035
2   0.902963    1052
3   0.892740    1053
4   0.883960    1055
5   0.895695    1056
6   0.920094    1059
7   0.902180    1038
8   0.867812    1039
9   0.781832    1018
10  0.830535    1014
11  0.885860    1015
12  0.874479    1030
13  0.914643    1017
14  0.884393    1010
15  0.922708    1011
16  0.866077    1034
17  0.708985    1013
18  0.912033    1043
19  0.848867    1042
20  0.822940    1041
21  0.656201    1032
22  0.926243    1026
23  0.895595    1033
24  0.880544    1016
25  0.868875    1025
26  0.918082    1024
27  0.839553    1027
28  0.892296    1031
29  0.869572    1047
30  0.798049    1046
31  0.868768    1045
32  0.879650    1022
33  0.866679    1061
34  0.896972    1036
35  0.861692    1049
36  0.867824    1048
37  0.912950    1028
38  0.873283    1037
39  0.876548    1051
40  0.789824    1060
41  0.879428    1007
42  0.779304    1006
43  0.601010    1004
44  0.911359    1003
45  0.388373    1002
46  0.849532    1001
```

**Comment by charleygros** [comment]
Sara! Merci! :)

- For the issue, I think that I kind of know how to fix it. I will send you a new version to test it on this subject!

- Regarding the leaks and imperfections at the level of compression... Mmmm very helpful to know... Ok will come back to 3D conv' and hope that it could improve it... ...

Encore merci!

**Comment by SaraDupont** [comment]
Example of results: 
- "mean" result (DC=0.85)
https://drive.google.com/open?id=1Wk0RS3Rb32Auu8XIQOJugfMYCoKFUUdQ

- worst result (DC=0.39)
https://drive.google.com/open?id=1teKl5iVXM3Iu2oqE5IU_4gW4m8zrwZ97

-best result (DC=0.93)
https://drive.google.com/open?id=1zYs8iPsaSe564SM4NBIALLjJ1d3cjlBN

**Comment by charleygros** [comment]
Mmmm ... the two first examples are not okay ...!
Merci 1000

**Comment by zougloub** [comment]
Closed by #1706
